frontend/package.json

```diff
@@ -80,6 +80,7 @@
     "@types/react-dom": "^16.9.14",
     "@types/react-paginate": "^7.1.1",
     "@types/react-test-renderer": "^17.0.1",
+    "@vercel/ncc": "^0.34.0",
```

Comment:
I'll remove the `ncc` patch after [PR](https://github.com/vercel/ncc/pull/914/files) in its repo get merged and the package is published.